lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -109,6 +112,11 @@ module Constants
     config_param :vm_id, :string, :default => nil
     config_param :vm_name, :string, :default => nil
 
+    # Set values from JSON payload with this key to the "trace" LogEntry field.
+    config_param :trace_key, :string, :default => DEFAULT_TRACE_KEY
+    # Whether to also keep the trace key/value in the payload.
+    config_param :keep_trace_key, :bool, :default => false
```

Comment:
I would guess it's mostly useful if a customer were already annotating their logs with traces, and wanted to keep their jsonPayload consistent, but also wanted the trace LogEntry field to be set. In that case, they would also set the `trace_key` config field.

Because the default `trace_key` is specific to the Google cloud logging API, customers would need to annotate their logs intentionally with it with the purpose of having the LogEntry trace field set, and so having it retained in the jsonPayload in that case would just add extra weight to the log entries. So that to me indicates that `false` is a good default.